File: fixture/CSharp/input/Student.cs
public class Student {
	private String _name;

	public Student(String name) {
		_name = name;
	}

	public String getName() {
		return _name;
	}
	
	public static void main(String[] args) {
		Student[] students = new Student[2];
		students[0] = new Student("Tom");
		students[1] = new Student("Anna");
		
		for (int i = 0; i < 2; i++) {
			write(students[i].getName());
		}
		foreach (Student student in students) {
			write(student.getName());
		}
	}
}

File: fixture/JavaScript/input/student.js
var Student = function(name) {
    this.name = name;
    this.getName = function() {
        return this.name;
    };
};

var list = new Array(2);
list[0] = new Student("Tom");
list[1] = new Student("Anna");

var i = 0;
for(i = 0; i < 2; i++) {
   alert(list[i].getName());
}